### PR TITLE
Base: Update the man pages so pre-commit does not fail when checking markdown files

### PR DESCRIPTION
**Pull request by mjz19910**
Also fix the few spelling errors in the man pages

### CODE REVIEW

**Comment by awesomekling** [comment]
`man` is a program. The commit message category for man pages is usually `Base` since that's where they live.

File: Base/usr/share/man/man1/CharacterMap.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Name
 
-![Icon](/res/icons/16x16/app-character-map.png) Character Map
+![Icon](file:///res/icons/16x16/app-character-map.png) Character Map
```
Comment (IdanHo) [inline Base/usr/share/man/man1/CharacterMap.md:3]: This breaks `man.serenityos.org`. See: https://github.com/SerenityOS/serenity/pull/11965

Comment (mjz19910) [reply Base/usr/share/man/man1/CharacterMap.md:3]: Then we need to remove these from markdown-check (pre-commit is broken after #11965), right now i just chmod -x the pre-commit hook

Comment (bgianfo) [reply Base/usr/share/man/man1/CharacterMap.md:3]: How about we fix the markdown check to ignore    `/res` image links for man pages?

**Review by bgianfo — CHANGES_REQUESTED** [review]
We need to keep the man pages working.

**Comment by IdanHo** [comment]
This was already fixed in https://github.com/SerenityOS/serenity/commit/bb340f1272e4d2316d0285c664492ce6a0413fe5

**Pull request closed without merging**